source/renderer/app/components/wallet/send-form/WalletSendAssetsConfirmationDialog.js

```diff
@@ -196,9 +91,9 @@ export default class WalletSendAssetsConfirmationDialog extends Component<
       fields: {
         passphrase: {
           type: 'password',
-          label: this.context.intl.formatMessage(messages.passphraseLabel),
+          label: this.context.intl.formatMessage(getMessages().passphraseLabel),
```

Comment:
Nice work with extracting messages into a separate file 👍 
Still, I would prefer to keep the `messages.passphraseLabel` format and add declare `const messages = getMessages();` on top.